components/web/php/php-7_0/Makefile

```diff
@@ -226,3 +228,5 @@ REQUIRED_PACKAGES += text/aspell
 REQUIRED_PACKAGES += text/tidy
 REQUIRED_PACKAGES += web/curl
 REQUIRED_PACKAGES += web/php-70/php-cli
+REQUIRED_PACKAGES += system/library/freetype-2
```

Comment:
Please, add mariadb-101/client and developer/icu above "# Auto-generated dependencies" line, as they are not found by pkgdepend